src/promptflow/promptflow/_sdk/_service/.gitignore

```diff
@@ -0,0 +1,2 @@
+openapitools.json
```

Comment:
merge this file with basic gitignore?

Reply:
this is not a global gitignore, so I prefer to leave it here